pandas/core/groupby/generic.py

```diff
@@ -63,7 +64,7 @@
 )
 from pandas.core.indexes.api import Index, MultiIndex, all_indexes_same
 import pandas.core.indexes.base as ibase
-from pandas.core.internals import BlockManager, make_block
+from pandas.core.internals import Block, BlockManager, make_block
```

Comment:
if Block is only used for annotations, can you put it in a `if TYPE_CHECKING`

Reply:
Ok.